.github/workflows/ci.yml

```diff
@@ -35,7 +35,9 @@ jobs:
       with:
         toolchain: stable
         override: true
-    - run: cargo test
+    - run: |
+        cargo test
+        cargo test --doc
```

Comment:
`cargo test` tests documentation as well. No?

Reply:
Yeah, I looked at this yesterday, as my understanding was the same and I couldn't make a rhyme on it. The confusion seems to be due to the `ignore` tag. If we tag the supposed-to-be-compiling code snippet with `no_run` instead we don't need the additional invocation with `--doc`. Will update. Thanks!